dev_tools/bash_scripts_test.py

```diff
@@ -15,18 +15,14 @@
 import os
 from typing import TYPE_CHECKING, Iterable
 
+import pytest
+
 from dev_tools import shell_tools
 
 if TYPE_CHECKING:
     import _pytest.tmpdir
 
 
-def only_on_posix(func):
-    if os.name != 'posix':
-        return None
```

Comment:
instead of removing this, you can turn this into a wrapper for pytest skipif fixture. I'm not sure exactly how to do it? maybe `return pytest.skip(reason)`

Reply:
I'm actually going to punt this change from this PR to another time :)